src/leibniz.f90

```diff
@@ -1,30 +1,18 @@
 program Leibniz
-    use, intrinsic :: iso_fortran_env, only: output_unit, error_unit, iostat_end, int64, real64
+    use, intrinsic :: iso_fortran_env, only: output_unit, real64
     implicit none
-    integer(int64) :: i, file_unit, rc
-    integer(int64) :: rounds;
-    real(real64) :: pi0, pi1, pi
+    integer :: i, file_unit, rc, rounds
+    real(real64), parameter :: x = -1
+    real(real64) :: pi
  
     open (action='read', file="rounds.txt", iostat=rc, newunit=file_unit)
-    read (file_unit, *, iostat=rc) rounds
+    read (file_unit, '(i20)', iostat=rc) rounds
     close (file_unit)
     
-    pi0 = 0.0_real64
-    pi1 = 0.0_real64
-
-    do i=0, rounds, 4
-        pi0 = pi0 + term(i) - term(i+1)
-        pi1 = pi1 + term(i+2) - term(i+3)
+    pi = 1
+    do i=1, rounds, 4
+        pi = pi + x / (2 * i + 1) - x / (2 * i + 3) + x / (2 * i + 5) - x / (2 * i + 7)
```

Comment:
eh?

Reply:
What does manual unrolling mean?

Reply:
idk, up to you to decide what is considered "fair" and "apple to apple" I guess

Reply:
I guess in the end it is 😅 
I think a good way to judge changes like that is how much impact do they have?
On the other hand, I feel it's increasingly more about how much attention a language gets to improve the implementation, so this could also start a trend to introduce manual unrolling in other implementations. 🤔

Reply:
> I feel it's increasingly more about how much attention a language gets to improve the implementation

then you might want to keep track of how much code each language is spending because at some point someone can give you a hand-optimized assembly code that runs the fastest

Reply:
If they do that by adding it as a new language, I would be fine with that. 😆